apps/mark-scan/frontend/src/index.tsx

```diff
@@ -23,15 +23,6 @@ const screenReader = new AriaScreenReader(
     : new SpeechSynthesisTextToSpeech(memoize(getUsEnglishVoice))
 );
 
-// Disable hot reloading because VxMarkScan hardware lacks resources to support it
```

Comment:
Would be nice to retain this comment in the `vite.config.ts` file. Also, maybe only disable if you can tell you're actually using the VxMarkScan hardware?

Reply:
Comment and environment variable to toggle hot reloading added in https://github.com/votingworks/vxsuite/pull/3643/commits/eb995df1a945bb6e1459b9b05f773ec8f4468f12

I'm not currently in front of the hardware so I wasn't able to play around with detecting the hardware. Bikeshedding on variable naming: I went back and forth on whether this should be an `ENABLE` or `DISABLE` variable, but since hot reloading is the default for all apps, it felt right to have a variable for disabling one specific app. Furthermore I don't love how single-app config exists in the root `.env`, but AFAIK we don't have project-level env files.